docs/application/native/guides/compliations/watchface-complication-provider.md

```diff
@@ -0,0 +1,247 @@
+one complication provider id.
+
+```xml
+<support-type>
+</support-type>
+```
+
+And, the specific default value is mandatory depending on each support types:
+
+| Support types | Mandatory default values | Optional default values |
+|---------------|--------------------------|---------------------------|
+| `<short-text-type>` | `<default-short-text>` | `<default-title>` |
+|                     |                        | `<default-icon>` |
+|                     |                        | `<default-extra-data>` |
+| `<long-text-type>` | `<default-long-text>` | `<default-title>` |
+|                    |                       | `<default-icon>` |
+|                    |                       | `<default-extra-data>` |
+| `<ranged-value-type>` | `<default-value>` | `<default-title>` |
+|                       | `<default-min>` | `<default-short-text>` |
+|                       | `<default-max>` | `<default-icon>` |
+|                       |                   | `<default-extra-data>` |
+| `<icon-type>` | `<default-icon>` | `<default-extra-data>` |
+| `<image-type>` | `<default-image>` | `<default-extra-data>` |
+| `<time-type>` | `<default-hour>` | `<default-short-text>` |
+|               | `<default-minute>` | `<default-extra-data>` |
+|               | `<default-second>` |         |
+
+
+`<label>` is the name of the complication provider.
+If it is necessary to support the multiple language, **multiple language** tag must be provided:
+
+```xml
+<label>MyComplication</label>
+<label xml:lang="en-us">EngComplication</label>
+```
+
+`<period>` is used to update complication provider data automatically, and in unit seconds:
+
+```xml
+<period>60</period>
+```
+
+In this case, the complication will send data update request to the complication provider every 60 seconds.
+
+`<support-event>` is the type of event that can be supported. It is possible to set **tap**, **double-tap**
+events or both.
+
+And it is possible to add `<privileges>` to restrict getting the complication data.
+In this case, watchface must add the specific privileges to get the complication data in `tizen-manifext.xml`:
+
+
+```xml
+<complication provider-id="mycomplicationid" setup-appid ="org.tizen.watch_setting" trusted="true">
+	<support-type>
+		<short-text-type>
+			<default-short-text>70</default-short-text>
+		</short-text-type>
+		<ranged-value-type>
+			<default-current>70</default-current>
+			<default-min>0</default-min>
+			<default-max>100</default-max>
+		</ranged-value-type>
+	</support-type>
+	<period>60</period>
+	<label>MyComp</label>
+	<label xml:lang="en-us">EngLabel</label>
+	<support-event>
+		<event>tap</event>
+		<event>double-tap</event>
+	</support-event>
+	<privileges>
+		<privilege>http://tizen.org/privilege/alarm.get</privilege>
+		<privilege>http://tizen.org/privilege/alarm.set</privilege>

```

Comment:
Please add:

## Related Information
- Dependencies
  - Tizen 5.0 and Higher for Wearable